pkg/plugins/profile/pd_profile_handler.go

```diff
@@ -21,9 +25,37 @@ const (
 	prefill = "prefill"
 )
 
+type pdProfileHandlerParameters struct {
+	prefix.Config
+	Threshold int `json:"threshold"`
+}
+
 // compile-time type assertion
 var _ framework.ProfileHandler = &PdProfileHandler{}
 
+// PdProfileHandlerFactory defines the factory function for the PdProfileHandler
+func PdProfileHandlerFactory(name string, rawParameters json.RawMessage, _ plugins.Handle) (plugins.Plugin, error) {
+	parameters := pdProfileHandlerParameters{
+		Config: prefix.Config{
+			HashBlockSize:          prefix.DefaultHashBlockSize,
+			MaxPrefixBlocksToMatch: prefix.DefaultMaxPrefixBlocks,
+			LRUCapacityPerServer:   prefix.DefaultLRUCapacityPerServer,
+		},
```

Comment:
does it make sense that prefix config appear on profile handler plugin?
shouldn't it appear on prefix plugin only?
something with this configuration looks not natural..

Reply:
This configuration is from the previous change to eliminate our own Prefix Scorer